htmresearch/frameworks/layers/l2_l4_inference.py

```diff
@@ -176,6 +180,15 @@ def __init__(self,
     @param   L4RegionType (string)
              The type of region to use for L4
 
+    @param   networkType (string)
```

Comment:
This has been addressed, but the comment is not correctly showing as outdated due to its scope.